.skiff/kubeconfig.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+set -e
+
+#
+# Script that generates the appropriate kubernetes manifest given the
+# provided parameters.
+#
+# This is intentionally simple. If we end up doing complex things, we should
+# likely embrace jsonnet or another community accepted solution for templated
+# kubernetes configuration files.
+#
+
+dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+usage() {
+  echo ""
+  echo "Usage:"
+  echo "  ./kubeconfig.sh ENV IMAGE"
+  echo ""
+  echo "  ENV   the environment identfier that indicates what configuration file to include, i.e. app.dev.config.sh"
+  echo "  IMAGE the docker image name to deploy"
+  echo ""
+}
+
+# Validate arguments
+ENV=$1
+if [[ -z "$ENV" ]]; then
+  (>&2 echo "Error: no ENV specified.")
+  usage
+  exit 1
+fi
+
+IMAGE=$2
+if [[ -z "$IMAGE" ]]; then
+  (>&2 echo "Error: no IMAGE specified.")
+  usage
+  exit 1
+fi
+
+# Validate required environment variables
+if [[ -z "$REPO_NAME" ]]; then
+  (>&2 echo "Error: the REPO_NAME environment variable must be set.")
+  usage
+  exit 1
+fi
+
+# Load the environment specific config
+source "$dir/config-$ENV.sh"
```

Comment:
@schmmd Did you add the configs per env? You need one for staging and prod, and then you need to set the `_ENV` var in the created trigger.